dashboard/app/views/devise/registrations/edit.html.haml

```diff
@@ -131,7 +131,8 @@
     }
   }
 
-- if Experiment.enabled?(user: current_user, experiment_name: 'ai-rubrics')
+-# Skip this section as ai-rubrics experiment needs to be unit aware
+-# if Experiment.enabled?(user: current_user, experiment_name: 'ai-rubrics')
```

Comment:
is the goal to hide the ai setting for all users? if so I think the rest of this block through line 145 also needs to be commented out, but I could certainly be missing something

Reply:
yes! it weirdly seemed to work by just commenting out the first line and the syntax highlighting, plus [this failing test](https://app.saucelabs.com/tests/5babb2c0067f43aba20302b8754e4c53#24) seems to support that

I can comment out the rest to make it clearer though if